src/dbus.c

```diff
@@ -856,13 +916,16 @@ gboolean dbus_cb_dunst_Properties_Set(GDBusConnection *connection,
                 g_dbus_connection_emit_signal(connection,
                                               NULL,
                                               object_path,
-                                              "org.freedesktop.DBus.Properties",
+                                              PROPERTIES_IFAC,
                                               "PropertiesChanged",
                                               g_variant_new("(sa{sv}as)",
                                                             interface_name,
                                                             builder,
                                                             invalidated_builder),
                                               NULL);
+
+                g_clear_pointer(&builder, g_variant_builder_unref);
+                g_clear_pointer(&invalidated_builder, g_variant_builder_unref);
```

Comment:
I found a few memory leaks.

Reply:
Thanks, that leak existed for more than 2 years! Although it only leaked memory every time you paused dunst.